### PR TITLE
add wiki sprint 4

### PR DESCRIPTION
**Pull request by barmettlerl**

### CODE REVIEW

File: wiki/Sprint-4.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Sprint 2
```
Comment (alexandrepiveteau) [inline wiki/Sprint-4.md:1]:
```suggestion
# Sprint 4
```

**Review by alexandrepiveteau — CHANGES_REQUESTED** [review]
I suggest you rename the "mood board" with a prefix or suffix that indicates that it belongs to the Sprint 4.

File: wiki/Sprint-4.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Matthieu Burguburu
+
+My main task for this sprint was to finish the design of the "Prepare game" screen on Figma, and implement both its UI and logic. My time estimates for these tasks were pretty good, but I took some extra unplanned time to implement a feature closely-related to this but which was slightly misplanned. However, this was nothing major and did not hinder me in my main task.
+
+Additionally, I had to finish increasing the coverage on a feature from last sprint. This should not have taken much time, but it lead to some refactoring and was a bit longer than expected.
+
+Overall, I think this 2 week sprint went quite well and did not make it harder to deliver my features than previous shorter sprints.
+
+## Chau Ying Kot (Scrum Master)
```
Comment (KurohanaJuri) [inline wiki/Sprint-4.md:22]: I'm not the scrum master for this sprint ;)

Comment (barmettlerl) [reply wiki/Sprint-4.md:22]: oobs sorry

File: wiki/Sprint-4.md (hunk 1 above)
Comment (barmettlerl) [reply wiki/Sprint-4.md:1]: thx

**Review by BadrTad — APPROVED** [review]

**Review by KurohanaJuri — APPROVED** [review]

**Review by matt989253 — APPROVED** [review]

**Review by alexandrepiveteau — APPROVED** [review]

**Pull request merged**